Dockerfile

```diff
@@ -9,6 +9,9 @@ LABEL org.opencontainers.image.source="https://github.com/giovtorres/slurm-docke
 ARG SLURM_TAG=slurm-22-05-4-1
 ARG GOSU_VERSION=1.11
 
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/Rocky-* && \
+    sed -i 's|#baseurl=|baseurl=|' /etc/yum.repos.d/Rocky-*
```

Comment:
Might want to anchor for the beginning of a line here.

Reply:
done